server_src/vrpn.cfg

```diff
@@ -2211,18 +2211,20 @@
 ################################################################################
 # Trivisio ColibriAPI support.  This is an inertial tracker that gives orientation
 # information, no position (the tracker reports (0, 0, 0) for position).
-# Use automatically created current.conf as a template for configuration file.
 # If wireless Colibri do not answer, dongle scans for them automatically (20 seconds).
 # Wireless Colibri must be in scanning mode (button pushed for 2 seconds,
 # LED flashing fast).
 #
 # Arguments:
 #	char	name_of_this_device[]
-#	char	conf_file_name[] (xml-file with sensor network configuration
-#                               or * to use all connected devices.)
-#	int     Hz (Update rate)
-#	int     report_a_w (0 = orientation only, 1 = orientation + angular
-#			    velocity + acceleration)
+#	char	conf_file_name[] : xml-file with sensor network configuration.
+#               Use Colibri GUI to create the file. Settings must be set
+#               and saved in sensors' non-volatile memory using Colibri GUI.
+#               * - to use all connected devices. Default settings will be
```

Comment:
Looks like a good approach.  How does the character "*" here produce a NULL name in the constructor?  Do you need to commit changes made to the vrpn_Generic_Server.C file as well?